### ISSUE TITLE
scrollNode.scrollTo is not a function 

### ISSUE THREAD

**Issue by bpmct** [netguru/sticky-parallax-header#91]
I did some searches and it looks like this is specifically an Android issue that comes up. However, I'm not familiar enough with React Native to fix this...

![7Tr1qpRvMI](https://user-images.githubusercontent.com/22407953/81635219-907a5c80-93c5-11ea-83f0-254531240d30.gif)

```
_this.scrollView.scrollTo is not a function. (In '_this.scrollView.scrollTo({
            x: offset,
            y: 0,
            animated: true
          })', '_this.scrollView.scrollTo' is undefined)

Stack trace:
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:17074:12 in commitLifeCycles
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:20002:23 in commitLayoutEffects
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:307:15 in invokeGuardedCallbackImpl
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:531:36 in invokeGuardedCallback
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:19768:10 in commitRootImpl
  [native code]:null in commitRootImpl
  node_modules\scheduler\cjs\scheduler.development.js:643:23 in unstable_runWithPriority
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:19590:4 in commitRoot
  [native code]:null in commitRoot
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:18709:28 in runRootCallback
  [native code]:null in runRootCallback
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:5642:32 in runWithPriority$argument_1
  node_modules\scheduler\cjs\scheduler.development.js:643:23 in unstable_runWithPriority
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:5638:22 in flushSyncCallbackQueueImpl
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:5627:28 in flushSyncCallbackQueue
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:18796:28 in batchedUpdates$1
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:2709:30 in batchedUpdates
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:2794:17 in batchedUpdates$argument_0
  node_modules\react-native\Libraries\Renderer\implementations\ReactNativeRenderer-dev.js:2870:28 in receiveTouches
  node_modules\react-native\Libraries\BatchedBridge\MessageQueue.js:436:47 in __callFunction
  node_modules\react-native\Libraries\BatchedBridge\MessageQueue.js:111:26 in __guard$argument_0
  node_modules\react-native\Libraries\BatchedBridge\MessageQueue.js:384:10 in __guard
  node_modules\react-native\Libraries\BatchedBridge\MessageQueue.js:110:17 in __guard$argument_0
  [native code]:null in callFunctionReturnFlushedQueue
```

**Comment by qoquma** [comment]
I have same problem

**Comment by IdaszakDaniel** [comment]
@bpmct @qoquma What version of React Native are you using?

**Comment by theisgroenbech** [comment]
I have the same issue running: Expo 37.0.3 (which uses React Native 0.61 internally)

**Comment by qoquma** [comment]
> @bpmct @qoquma What version of React Native are you using?

"version": "0.61.5"

**Comment by SrAnthony** [comment]
Using `scrollNode.getNode().scrollTo` seems to work

**Comment by IdaszakDaniel** [comment]
`.getNode()` was removed in 0.2.0 version of the library, as in [ 0.62 version of React Native it's deprecated](https://reactnative.dev/blog/2020/03/26/version-0.62). If you can't upgrade to 0.62, use 0.1.0 version or use [patch-package](https://www.npmjs.com/package/patch-package) to revert [this commit](https://github.com/netguru/sticky-parallax-header/commit/ee5d224dc54055ca1948d4d4307593c500ee1697).

**Comment by bpmct** [comment]
> `.getNode()` was removed in 0.2.0 version of the library, as in [ 0.62 version of React Native it's deprecated](https://reactnative.dev/blog/2020/03/26/version-0.62). If you can't upgrade to 0.62, use 0.1.0 version or use [patch-package](https://www.npmjs.com/package/patch-package) to revert [this commit](https://github.com/netguru/sticky-parallax-header/commit/ee5d224dc54055ca1948d4d4307593c500ee1697).

Worked great. Expo does not naturally support 0.62 but I was able to apply the patch :)

**Comment by ChrisCruze** [comment]
I upgraded React Native to 0.62, but then my app load at all, so I had to downgrade React Native to 0.615 and I get the below error from the app
<img width="374" alt="Screenshot 2020-05-22 11 10 44" src="https://user-images.githubusercontent.com/34425825/82682196-e4364280-9c1c-11ea-99ab-32888a296291.png">

See below for current configuration: 
   "react": "~16.9.0",
    "react-dom": "~16.9.0",
    "react-native": "0.61.5",
    "expo": "^37.0.12",
    "@expo/vector-icons": "^10.0.0",
    "react-native-sticky-parallax-header": "^0.2.1"

**Comment by PaulHaze** [comment]
> Using `scrollNode.getNode().scrollTo` seems to work

Hi, I am also getting this error. 
![Screen Shot 2020-05-25 at 9 13 19 am](https://user-images.githubusercontent.com/20569604/82767089-1ab3cd80-9e68-11ea-9d9d-d6be826ebe72.png)

Where exactly do I place the `scrollNode.getNode().scrollTo` code? do I replace `_this.scrollView.scrollTo` with `_this.scrollView.scrollNode.getNode().scrollTo`? or replace the whole thing with `.scrollNode.getNode().scrollTo, because I have tried both and neither seem to work. 

This package is _exactly_ what I am looking for in a project I am working on so it would be great to get this resolved somehow....

**Comment by akug17** [comment]
@PaulHaze This seems to be working fine

**Comment by bittu1028** [comment]
Running your example for tabbed header has same error ? this issue should not be closed i am using react native v 0.6.6

**Comment by partitect** [comment]
I m using expo sdk 37 and
 "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz"
if i upgrade manually rn 0.62, app crashed.
when i use v 0.1.0 dont change details header content.
how do i use v0.2.0 in expo sdk 37

**Comment by Crael94** [comment]
Having the same issue... Does someone have a fix?

**Comment by Kleinhummel** [comment]
> @PaulHaze This seems to be working fine

Yes, but can someone of the contributors of this package include this and provide a new version of the package? I don't think it is recommended that everyone using this package has to fix this on his own...

**Comment by Kleinhummel** [comment]
> > Using `scrollNode.getNode().scrollTo` seems to work
> 
> Hi, I am also getting this error.
> ![Screen Shot 2020-05-25 at 9 13 19 am](https://user-images.githubusercontent.com/20569604/82767089-1ab3cd80-9e68-11ea-9d9d-d6be826ebe72.png)
> 
> Where exactly do I place the `scrollNode.getNode().scrollTo` code? do I replace `_this.scrollView.scrollTo` with `_this.scrollView.scrollNode.getNode().scrollTo`? or replace the whole thing with `.scrollNode.getNode().scrollTo, because I have tried both and neither seem to work.
> 
> This package is _exactly_ what I am looking for in a project I am working on so it would be great to get this resolved somehow....
> 
> EDIT: After tinkering around some more I have found this solution seems to work so I'm posting it here in case it can help others. Basically I went to the StickyParallaxHeader.js file and found ALL occurences of`scrollTo()` I could find and inserted '`getNode().` in front of them. Like this:
> 
> ![Screen Shot 2020-05-25 at 9 46 50 am](https://user-images.githubusercontent.com/20569604/82767702-ce1ec100-9e6c-11ea-8dd8-dd97091068cc.png)
> 
> To get the tabs example working I also went to ScrollableTabView.js found in `react-native-sticky-parallax-header/src/components/ScrollableTabView/` and inserted `getNode().` in front of `scrollTo()` found in the `goToPage` function:
> 
> ![Screen Shot 2020-05-25 at 9 44 24 am](https://user-images.githubusercontent.com/20569604/82767805-919f9500-9e6d-11ea-938d-e343aad189b2.png)
> 
> So far this SEEMS to work, but its far from battle tested so YMMV. I'm using this on Expo 37.
> 
> Hope that is of some help....

I created a pull request that did exactly that: #100

**Comment by pwysowski** [comment]
Issue was fixed in PR #106
